javascript/adonisjs6-http/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "module",
+  "dependencies": {
+    "@adonisjs/application": "^8.0.4",
+    "@adonisjs/encryption": "^6.0.1",
+    "@adonisjs/events": "^9.0.1",
+    "@adonisjs/http-server": "^7.0.3",
+    "@adonisjs/logger": "^6.0.1"
```

Comment:
```suggestion
  "dependencies": {
    "@adonisjs/application": "*",
    "@adonisjs/encryption": "*",
    "@adonisjs/events": "*",
    "@adonisjs/http-server": "~7.0.3",
    "@adonisjs/logger": "*"
```

Reply:
Why is that needed?

Reply:
because we have a bot that take care of updates, and having PR each time a minor is update (for events for example) is like a too heavy task for me

also, it is, imho, a good way to do, to see for any breaking change

Reply:
> we have a bot that take care of updates

But semver is here for that reason. There is no need for a PR. Any patch and minor release will be installed when updating/installing the dependencies.

With a `*`, if we release a major release, it will be installed and maybe break something.

Reply:
Yes, I understand. Then their is 2 solutions :
+ Leave `*` and ping you when a breaking change is detected (each release)
+ Use semver and ping you on each minor update

I understand the concerns, but I can not maintain all versions myself for each frameworks

Reply:
PS : I use `*` strategy and I have not some many breakings

Reply:
> Use semver and ping you on each minor update

If we are using `^` it will be only each **major** update, using `~` in the semver condition will not allow minor update, but it was `^`  before.

Reply:
Yes, feel free to use the carret then, but I'll ping you when updates

Reply:
If you are comfortable with that, I will push it for the next update

Reply:
Yes, I am good with that, and I believe it is better! 👍🏻